service-account/aws/variables.tf

```
@@ -0,0 +1,43 @@
+variable "prefix" {
+  description = "Prefix to use for service account related resources"
+  type        = string
+}
+
+# Tags
+variable "tags" {
+  description = "Tags for resource"
+  type        = map(string)
+  default     = {}
+}
+
+variable "namespace" {
+  description = "Namespace of ArmoniK service account related resources"
+  type        = string
+  default     = "armonik"
+}
+
+variable "service_account_name" {
```

You are in the service-account module, there is no need to prefix your parameters with service_account

```suggestion
variable "name" {
```